components/Carousel/styles.js

```diff
@@ -16,7 +19,7 @@ const styles = StyleSheet.create({
     flex: 1,
     borderRadius: 5,
     backgroundColor: 'lightblue',
-    marginBottom: Platform.select({ ios: 0, android: 1 }), //handle rendering bug.
+    // marginBottom: Platform.select({ ios: 0, android: 1 }), //handle rendering bug.
```

Comment:
Curious about how Platform.select is working here.

Reply:
Agreeing with Chantal... is the commented code indicating that this will handle the rendering bug or this is where the bug is living right now?